File: Algorithm10.py
result = 0
for a in range(2000001):
    count = 0
    for b in range(1, a+1):
        if a % b == 0:
            count += 1
        if count == 3:
            break
    if count == 2:
        result += a
        print('%s || %s'%(result, a))
print(result)

File: Algorithm10.java
package com.ajh.practice;

public class Algorithm10 {
	int result;
	Algorithm10(){
		for(int i = 0; i > 2000001; i++) {
			int count = 0;
			for(int j =1; j > i+1; j++) {
				if (i % j == 0)
					count += 1;
				if (count == 3)
					break;
			if (count == 2)
				result += i;
			}
		}
	}public static void main(String[] args) {
		Algorithm10 a = new Algorithm10();
		System.out.println(a.result);
	}
}
